src/Calculator/Resources/en-US/Resources.resw

```diff
@@ -562,7 +562,7 @@
     <comment>Sub-string used in automation name for each bit in bit flip</comment>
   </data>
   <data name="63" xml:space="preserve">
-    <value>63rd </value>
```

Comment:
Thanks for tackling this one! I think the most significant bit is a bit more complicated to handle. It is the 63rd bit for qwords, but the 31st for dwords, 15th for words, and 7th for bytes.

Reply:
Thanks for keeping me honest @grochocki. Doh, face palm, of course here. It does make the issue more interesting. I'll take a look and figure out how to resolve that.

I'm new to the PR process. Is there anything I should do.

Reply:
No problem! Easy to miss since qwords are the default. Check out our [documentation](https://github.com/Microsoft/calculator/blob/master/CONTRIBUTING.md#making-changes-to-the-code) for more info on making code changes and PR process.